modules/web-api.py

```diff
@@ -209,6 +204,7 @@ def main():
     parser.add_argument('-tenant', dest='tenant', required=True, type=str, default='EGI', help='tenant to check. Default EGI')
     parser.add_argument('-rtype', dest='rtype', required=True, type=str, default='ar', help='status or ar (default ar)')
     parser.add_argument('-token', dest='token', required=True, type=str, default='test_token', help='authentication token')
+    parser.add_argument('-unused-reports', dest='unusedreports', required=False, type=str,help='Add unused reports from API as a string separated by comma')
```

Comment:
Add ``nargs='+'`` so argument will swallow everything specified to it into ``args.unusedreports`` which will be of type list. That way you don't need to imply comma separated list of reports enclosed in quotes. You can just simply say: ``--unused-reports Report Report2``. List should be also easier to handle further in code.
Also, multicharacter argument are longoptions (see getopt(1)) and those should start with ``'--'``